src/config.h.in

```diff
@@ -21,30 +21,14 @@
 #ifndef DOSBOX_CONFIG_H
 #define DOSBOX_CONFIG_H
 
-/* Version and branding tweaks
-*/
-
-// Name of project, lower-case without spaces
-#define CANONICAL_PROJECT_NAME "@project_name@"
-
-// Emulator Semantic Version (MAJOR.MINOR.PATCH), incremented as follows:
-//  - MAJOR version when you make incompatible API changes
-//  - MINOR version when you add functionality in a backwards compatible manner
-//  - PATCH version when you make backwards compatible bug fixes
-// Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
-// Ref: https://semver.org/
-
-#define VERSION "@version@"
-
-/* Strings to be returned by virtual drivers, etc.
-*/
-
-// Name of the emulator
-#define DOSBOX_NAME "DOSBox Staging"
-// Development team name
-#define DOSBOX_TEAM "The " DOSBOX_NAME " Team"
-// Copyright string
-#define DOSBOX_COPYRIGHT "(C) " DOSBOX_TEAM
+// This file is a template for the Meson build process.
+//
+// '@name@' placeholders are substituted with concrete values at the start of
```

Comment:
this throws a warning during build:

```log
../../meson.build:1296: WARNING: The variable(s) 'name' in the input file 'src/config.h.in' are not present in the given configuration data.
```

maybe escape it, e.g.:

```cpp
'@''name@'
```

Reply:
Rephrased it a bit, now the warning is gone.